content/minutes/triage-meeting/2019-09-26.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 2019.09.26
+type: docs
+---
+
+# 2019-09-26
+
+## Volunteers wanted
+
+- Help wanted: proc-macro param attrs dropping first attrs in impl fns [#64682](https://github.com/rust-lang/rust/issues/64682)
+
+- Help wanted: error: internal compiler error: unexpected panic: inconsistent resolution for a macro [#64803](https://github.com/rust-lang/rust/issues/64803)
+
+## Backport nominations
+
+- Fix #64744. Account for the Zero sub-pattern case. [#64748](https://github.com/rust-lang/rust/pull/64744)
+  - Approved for beta backport
+
+- Rustdoc render async function re-export [#64599](https://github.com/rust-lang/rust/pull/64599)
+  - Approved for beta backport
+
+## Working group sync
+
+### [wg-traits](https://rust-lang.github.io/compiler-team/working-groups/traits/)
+
+`wg-traits` is spinning back up and has a number of projects currently being worked on:
+
+  - More Chalk work
+  - Investigating lazy normalization
+  - Enabling trait upcasts
+  - Resolving `dyn Trait` coherence and well-formedness issues
+  - Work is progressing to allow stabilization of the `~` type
```

Comment:
Where can I find more about this type? Or does it mean `!`?

Reply:
That's a recent addition to the type system; it's pronounced "the typo type" 😄